data-protocols/ids/ids-api-configuration/src/main/java/org/eclipse/dataspaceconnector/ids/api/configuration/IdsApiConfigurationExtension.java

```diff
@@ -61,8 +66,12 @@ public void initialize(ServiceExtensionContext context) {
         }
         
         monitor.info(format("IDS API will be available at [path=%s], [port=%s].", path, port));
-        
-        context.registerService(IdsApiConfiguration.class, new IdsApiConfiguration(contextAlias, path));
+
+        var idsWebhookAddress = context.getSetting(IDS_WEBHOOK_ADDRESS, DEFAULT_IDS_WEBHOOK_ADDRESS);
+        var webhookPath = path + (path.endsWith("/") ? "data" : "/data");
+        idsWebhookAddress = idsWebhookAddress + webhookPath;
```

Comment:
nit:
```suggestion
        var idsWebhookAddress = context.getSetting(IDS_WEBHOOK_ADDRESS, DEFAULT_IDS_WEBHOOK_ADDRESS) + webhookPath;
```
and removing line 70